types/trace/trace.go

```diff
@@ -31,84 +31,19 @@ type TraceeEvent struct {
 	Args                []Argument `json:"args"` //Arguments are ordered according their appearance in the original event
 }
 
-type Stats struct {
-	EventCount  int
-	ErrorCount  int
-	LostEvCount int
-	LostWrCount int
-	LostNtCount int
-}
```

Comment:
Why removing Stats from here?

Reply:
Because it was left here by mistake, the actual stats are exported in the `metrics` packages.